modules/phase_field/test/tests/electrochem_sintering/tests

```diff
@@ -0,0 +1,7 @@
+[Tests]
+  [ElectrochemicalSintering]
+    type = 'Exodiff'
+    input = 'ElectrochemicalSintering_test.i'
+    exodiff = 'ElectrochemicalSintering_test_out.e'
+  []
```

Comment:
I think you need to add `design`, `issue` staff, correct?

Reply:
Yes you're right, I will fix it. Actually I'm surprised the test system didn't throw an error with that not being there